### ISSUE TITLE
error Couldn't find any versions for "@webassemblyjs/helper-numbers" that matches "1.11.3"

### ISSUE THREAD

**Issue by coderaiser** [xtuc/webassemblyjs#1132]
Cannot install webassemblyjs

```
error Couldn't find any versions for "@webassemblyjs/helper-numbers" that matches "1.11.3"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Couldn't find any versions for "@webassemblyjs/helper-api-error" that matches "1.[11](https://github.com/coderaiser/putout/runs/7171211450?check_suite_focus=true#step:5:12).3"
    at MessageError.ExtendableBuiltin (/usr/local/lib/node_modules/yarn/lib/cli.js:721:66)
```

https://github.com/coderaiser/putout/runs/7171211450?check_suite_focus=true

**Comment by andersk** [comment]
5fd2425602b752576bbe8089c343d5d70ebc861c bumped all of the versions to 1.11.4 but left (nearly) all of the dependencies stuck at 1.11.3. Some kind of Lerna accident?

**Comment by coderaiser** [comment]
There is no such version as `@webassemblyjs/helper-numbers@1.11.3` in https://www.npmjs.com/package/@webassemblyjs/helper-numbers

![image](https://user-images.githubusercontent.com/1573141/177215464-d72f5d7c-1d65-4bf7-9afc-2cdf1bd6c4c2.png)

So it's unpublished, or there was never such a version 🤷‍♂️

**Comment by bpowers** [comment]
cc @xtuc -- I ran into this while trying to bump WebPack's `webassemblyjs/*` dependencies in https://github.com/xtuc/webassemblyjs/pull/1117 .  I imagine the thing to do is to either hit release on the rest of the modules for 1.11.4, or bump the version to 1.11.5 and try to bump everything.

**Comment by xtuc** [comment]
I upgraded lerna to support npm OTP but looks like they don't support the flags: `--force-publish --exact` anymore :confused:

**Comment by hsjobeki** [comment]
> cc @xtuc -- I ran into this while trying to bump WebPack's `webassemblyjs/*` dependencies to pull in #1117 . I imagine the thing to do is to either hit release on the rest of the modules for 1.11.4, or bump the version to 1.11.5 and try to bump everything.

Could this be solved ?

many depending toolchains broke on your changes.

**Comment by LadenxxxxD** [comment]
same issue, can't install my project dependencies now : )

**Comment by RobLoach** [comment]
Same with https://www.npmjs.com/package/@webassemblyjs/helper-api-error .

**Comment by xtuc** [comment]
I manually published version 1.11.5 which should fix this issue. Can you please confirm it's working on your side?
